python/pyspark/sql/dataframe.py

```diff
@@ -5503,13 +5502,15 @@ def drop(self, *cols: "ColumnOrName") -> "DataFrame":  # type: ignore[misc]
         | 16|    85|
         +---+------+
 
-        Can not drop col('name') due to ambiguous reference.
-
+        Example 5: Can not drop col('name') due to ambiguous reference.
+        >>> from pyspark.sql.functions import col
         >>> df3.drop(col("name")).show()
         Traceback (most recent call last):
         ...
         pyspark.errors.exceptions.captured.AnalysisException: [AMBIGUOUS_REFERENCE] Reference...
 
+        Example 6: Can not find a column matching the expression "a.b.c".
+        >>> from pyspark.sql.functions import lit
```

Comment:
let's follow the `quick start` example in this PR:

`from pyspark.sql import functions as sf`